### PR TITLE
Packaging improvements

### PR DESCRIPTION
**Pull request by GhostofGoes**
- Add a detailed description that will appear on the PyPI project page.
- Add python_requires, which helps pip determine compatibility when
installing the package (it doesn't use pypi classifiers)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
+#!/usr/bin/env python
```
Comment (tk0miya) [inline setup.py]: setup.py is not executable. So it seems shebang line is not needed.

**Review by tk0miya — COMMENTED** [review]
LGTM with nits.

**Comment by GhostofGoes** [comment]
Updated, should be good to go now.

**Review by tk0miya — APPROVED** [review]
LGTM!

**Comment by tk0miya** [comment]
Merged. Thank you for your contribution.

**Pull request merged**